app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileEditActivityTest.kt

```diff
@@ -313,7 +314,12 @@ class ProfileEditActivityTest {
         .inRoot(isDialog())
         .perform(click())
       testCoroutineDispatchers.runCurrent()
-      intended(hasComponent(ProfileListActivity::class.java.name))
```

Comment:
is this test case failing before this implementation?

Reply:
no it does'nt fail but we can't get the required screen

Reply:
Expected Screen after deleting profile | What we get 
--- | --- 
![Screenshot_1606578255](https://user-images.githubusercontent.com/54740946/100519799-2c33b480-31c0-11eb-8176-19ba7933f26c.png) | ![Screenshot_1606578255](https://user-images.githubusercontent.com/54740946/100519803-2dfd7800-31c0-11eb-98c2-5c4ea85ada2b.png)

Reply:
is it working correctly without RTl?

Reply:
@anandwana001 I don't get it, I ran it without RTL

Reply:
ah, I checked, it's working correctly with RTL too.